src/main/java/com/ait/lienzo/client/core/shape/wires/handlers/impl/WiresDockingControlImpl.java

```diff
@@ -157,6 +160,9 @@ private Point2D computeCandidateLocation(WiresShape shape) {
     }
 
     private Point2D getCandidateLocation(WiresShape shape) {
+        if (null == m_intersection) {
+            return null;
```

Comment:
Good catch, @jomarko,

Yes, I found one possible null pointer for this path:
`WiresShapeControlImpl#execute` -> `WiresDockingControlImpl#getCandidateLocation()` -> `WiresDockingControlImpl#getCandidateLocation(WiresShape)`.

If this path is not reachable so far, it can be in the future. @romartin, is it possible to protect us from it now and in the future?